src/jit/gentree.cpp

```diff
@@ -7122,7 +7106,8 @@ GenTreePtr GenTree::GetChild(unsigned childNum)
     printf("%c", (flags & GTF_CALL          ) ? 'C' : '-');
     printf("%c", (flags & GTF_EXCEPT        ) ? 'X' : '-');
     printf("%c", (flags & GTF_GLOB_REF      ) ? 'G' : '-');
-    printf("%c", (flags & GTF_ORDER_SIDEEFF ) ? 'O' : '-');
+    printf("%c", (flags & GTF_MORPHED       ) ? '+' :
+                 (flags & GTF_ORDER_SIDEEFF ) ? 'O' : '-');
```

Comment:
I will added a comment
 printf("%c", (flags & GTF_MORPHED       ) ? '+' :                 // First print '+' if GTF_MORPHED is set
                  (flags & GTF_ORDER_SIDEEFF ) ? 'O' : '-');      // otherwise print 'O' or '-'